src/weather.js

```diff
@@ -14,9 +14,18 @@ export class Weather {
 
   return  this.celsius;
 
+}
 
+async londonDescription() {
+  const response = await fetch('https://api.openweathermap.org/data/2.5/weather?q=London'+ '&appid='+process.env.API_KEY)
+  const json = await response.json();
+  const weatherDescription = json.weather[0].description;
+console.log(weatherDescription);
+  return weatherDescription;
 }
-  convert(temp) {
+
+
+convert(temp) {
```

Comment:
If you point your api call to:
`https://api.openweathermap.org/data/2.5/find?q=London,UK&units=metric`
this will provide you with the units in celsius. 
As a result, you won't need your convert method :)